neqo-transport/src/recovery.rs

```diff
@@ -97,6 +100,65 @@ impl RttVals {
     }
 }
 
+/// `SendProfile` tells a sender how to send packets.
+#[derive(Debug)]
+pub struct SendProfile {
+    limit: usize,
+    pto: Option<PNSpace>,
+    paced: bool,
+}
+
+impl SendProfile {
+    pub fn new_limited(limit: usize) -> Self {
+        // When the limit is too low, we only send ACK frames.
+        // Set the limit to `ACK_ONLY_SIZE_LIMIT - 1` to ensure that
+        // ACK-only packets are still limited in size.
```

Comment:
Yes, I know that our ACK sender doesn't pay attention to the space it has available, but it probably should.